src/nbl/video/CVulkanLogicalDevice.cpp

```diff
@@ -727,6 +727,76 @@ void CVulkanLogicalDevice::updateDescriptorSets_impl(const SUpdateDescriptorSets
     m_devf.vk.vkUpdateDescriptorSets(m_vkdev,vk_writeDescriptorSets.size(),vk_writeDescriptorSets.data(),vk_copyDescriptorSets.size(),vk_copyDescriptorSets.data());
 }
 
+void CVulkanLogicalDevice::nullifyDescriptors_impl(const std::span<const IGPUDescriptorSet::SDropDescriptorSet> drops, asset::IDescriptor::E_TYPE descriptorType)
+{
+    if (getEnabledFeatures().nullDescriptor)
+    {
```

Comment:
my preferred way of bailing is returning at the start, instead of nesting oneself in extra `{}`, style nitpick